app/utils/info.ts

```diff
@@ -0,0 +1,41 @@
+import { Alert } from 'react-native';
+
+import I18n from '../i18n';
+
+export const showErrorAlert = (message: string, title?: string, onPress = () => {}): void =>
```

Comment:
this titile is not required ?

Reply:
The component `Alert.alert` from  React Native, requires a title, but the message is optional. However, in our repository sometimes we pass only the first element, that is the message!

Reply:
WDYT?

Reply:
The point is that as you said, `title` is a mandatory property for `Alert`, we need to identify where this parameter is not passed and study a standard text.

Reply:
When you pass `null` or a empty string, the title will be hided. IMO we needed a default title for this cases. 
But let's discuss this another time with our team.